code/modules/mapping/map_specific_code/Tramstation.dm

```diff
@@ -1,6 +1,118 @@
 /obj/modular_map_root/tramstation
 	config_file = "strings/modular_maps/Tramstation.toml"
 
+//left of dorms on the upper floor, beside library
```

Comment:
Does having a desc variable on `/obj/modular_map_root` break stuff? I'm just wondering if it isn't worthwhile to add DM-side descriptions (that'll show up in the map editor) that are contextualized from the information in the TOML.

For example:
```dm
/obj/modular_map_root/tramstation/maintenance/foo
 desc = "Spawns in one of the following modules: "maintenance_foo_1.dmm", "maintenance_foo_2.dmm", "maintenance_foo_3.dmm"
```

I don't know how helpful this might be, I'm just trying to figure out the best way to have all the information be as easily accessible as possible to a new-comer. Up to you.